core/server/lib/members/index.js

```diff
@@ -207,6 +207,20 @@ module.exports = function MembersApi({
     httpHandler.staticRouter = staticRouter;
     httpHandler.apiRouter = apiRouter;
     httpHandler.memberUserObject = users;
+    httpHandler.updateSubscription = function (paymentConfig) {
```

Comment:
Not a big deal, but this method reads like it's an action on an object, rather than the api - might be better named as `reconfigure` or smth?

Reply:
Sure, wasn't sure about a nicer name, might need to reconfigure more bits in future, will name it `reconfigureSettings` for now. 👍